crates/bdk/src/wallet/utils.rs

```diff
@@ -110,6 +111,19 @@ impl<Pk: MiniscriptKey + ToPublicKey> Satisfier<Pk> for Older {
     }
 }
 
+// The Knuth shuffling algorithm based on the original Fisher-Yates method: [https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle]
```

Comment:
This is not a docstring, my suggestion:

```suggestion
/// The Knuth shuffling algorithm based on the original
/// (Fisher-Yates method)[https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle].
```

Reply:
Same comment as above

Reply:
Ok, then sorry about the nagging. It's fine as it is.